features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/utils/config/GlobalRumMonitorTestConfiguration.kt

```diff
@@ -9,19 +9,33 @@ package com.datadog.android.rum.utils.config
 import com.datadog.android.core.InternalSdkCore
 import com.datadog.android.rum.GlobalRumMonitor
 import com.datadog.android.rum.RumMonitor
+import com.datadog.android.rum._RumInternalProxy
 import com.datadog.android.rum.internal.monitor.AdvancedRumMonitor
 import com.datadog.tools.unit.extensions.config.MockTestConfiguration
 import fr.xgouchet.elmyr.Forge
 import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+@Suppress("TestFunctionName")
+internal abstract class InternalAdvancedRumMonitor : AdvancedRumMonitor {
```

Comment:
Like @xgouchet I do not understand why do you need this class : `InternalAdvancedRumMonitor` ?

Reply:
It seems like the `_getInternal` method is not visible in unit testing, unless explicitly implemented.

https://github.com/DataDog/dd-sdk-android/pull/2133#discussion_r1689429377

Reply:
yes because mockito is using Java under the hood and that method is not visible in Java if you annotate it with @JvmSynthetic :(. I don't like this but I guess we can't go around it.